X16 Reference - 03 - BASIC.md

```diff
@@ -164,11 +164,18 @@ Because of the differences in hardware, the following functions and statements a
 
 The BASIC interpreter also currently shares all problems of the C64 version, like the slow garbage collector.
 
+## Saving Files
+
+By default, you cannot automatically overwrite a file with SAVE, BSAVE, or OPEN. To overwrite a file, you must prefix the filename with `@:`, like this: `SAVE "@:HELLO WORLD"`. (`"@0:filename` is also acceptable.)
+
+This follows the Commodore convention, which extended to all of their diskette drives and third party hard drives and flash drive readers.
+
+Always confirm you have successfully saved a file with the DOS command (with no arguments). This will print either `00,OK,00,00` or a FILE EXISTS error if you forgot to prepend the @: prefix.
+
```

Comment:
The SAVE command in immediate mode already does this now.

Reply:
Updated docs with a routine to read the basic error channel data (left out the track and sector, since they're always 0,0 in CMDR-DOS) and stipulated that in Immediate/Direct mode SAVE now does an implicit DOS check.